src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Buldozer is work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Buldozer is stop work");

```

Comment:
The message 'Buldozer is stop work' is not informative enough and contains a grammatical mistake. It should be 'Bulldozer stopped its work' to clearly indicate that the Bulldozer has stopped working.